posts/indu-plp-part1/plp-part1.qmd

```diff
@@ -0,0 +1,217 @@
+---
+title: "PLP-Pipeline Series Part 1: From Research Question to Cohort Construction"
+description: "Kicking off the PLP-Pipeline blog series - how we define research questions and construct cohorts using OMOP CDM and Julia tools."
+author: "Kosuri Lakshmi Indu"
+date: "4/12/2025"
+bibliography: ./references.bib
+csl: ./../../ieee-with-url.csl
+toc: true
+engine: julia
+image: false
+categories:
+  - patient level prediction
+  - omop cdm
+  - observational health
+---
+
+# Introduction 👋
+
+Hi everyone! I’m **Kosuri Lakshmi Indu**, a third-year undergraduate student in Computer Science and an aspiring GSoC 2025 contributor. My interest in using data science for public health led me to the **JuliaHealth** community and, under the mentorship of Jacob S. Zelko, I began working on a project titled **PLP-Pipeline**. This project focuses on building modular, efficient tooling for Patient-Level Prediction (PLP) entirely in Julia, using the OMOP Common Data Model (OMOP CDM).
+
+In this post, I’ll walk through the first part of a three-part blog series documenting my work on building a Patient-Level Prediction (PLP) pipeline in Julia. Each post focuses on a different stage of the pipeline:
+
+1. **From Research Question to Cohort Construction (this post)**
+
+2. From Raw Clinical Data to Predictive Models
+
+3. Lessons Learned, Key Challenges, and What Comes Next
+
+In Part 1, we’ll start at the very beginning-formulating the research question, exploring the OMOP CDM, setting up the local database, and defining target and outcome cohorts using Julia tools. Whether you're a health researcher, a GSoC aspirant, or a Julia enthusiast, I hope this gives you a clear and accessible introduction to how observational health research can be made more composable, reproducible, and efficient using Julia.
+
+You can find my [**PLP-Pipeline Project Link Here**](https://github.com/kosuri-indu/PLP-Pipeline)
+
+[**LinkedIn**](https://www.linkedin.com/in/kosuri-indu/) | [**GitHub**](https://github.com/kosuri-indu/)
+ 
+# Background
+
+## What is Observational Health?
```

Comment:
More concrete details about Observational Health could be pulled from https://arxiv.org/pdf/2304.06504